### ISSUE TITLE
bug on kitkat 4.4.2

### ISSUE THREAD

**Issue by chuppito** [M66B/XPrivacy#2133]
Hello, I was serving in android 4.2.1 and it worked well. I changed my phone I kitkat on 4.4.2 I had to remove it because I have a lot of bug (sms notification that remains connected screen phone turns still on standby, call one of my contacts without be aware that, unable to play candy crush, and still other worries). Thank you for the work of this great tool I hope to use it again.

My old phone on jelly bean 4.2.1 : wiko stairway (the same as micromax canvas 4 A210)
My new phone on kitkat 4.4.2 : wiko highway signs (the same as micromax A290)

**Comment by M66B** [comment]
Please read this:https://github.com/M66B/XPrivacy#FAQ52

**Comment by chuppito** [comment]
Ok I'll try to explain my problems is that makes the google translation.
Xprivacy is a great tool which I used on my old phone. So I installed on my new phone and I know that it is he who create bugs me because since I removed the phone working again.
I use hangouts for receiving SMS and not the core application. When I xprivacy, the core application let me all notifications in the taskbar. I fixed this problem by removing the native application.
If I let this application and I remove xprivacy, the problem disappears.

When I drive, I put my phone on charge in developer mode, I checked the box so it does not go to sleep when it loads. It is very useful in the car when I start a streaming radio, the phone is on the radio and do not go to sleep. With Xprivacy, the phone goes into standby anyway. As soon as I remove Xprivacy, it stays on.

I wanted to test candy crush, xprivacy was installed, this game was singing on the first screen (orange screen if you know well that game) and was closing on. Since I no longer xprivacy, I could play it normally starts.
I said that I had given all rights to candy crush.

When I got a phone call and as soon as I hang up, if I'm not careful, the phone call any contact. Since I do not have Xprivacy I not what worries

The rest of the bugs, I do not remember, but if you have a solution for these problems, I will install again with great pleasure
